mmv1/third_party/terraform/tests/resource_alloydb_instance_test.go.erb

```diff
@@ -0,0 +1,98 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAlloydbInstance_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProvidersOiCS,
+		CheckDestroy: testAccCheckAlloydbInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAlloydbInstance_alloydbInstanceBasicExample(context),
+			},
+			{
+				ResourceName:            "google_alloydb_instance.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"cluster_id", "location", "instance_id"},
```

Comment:
Same here, these fields should be set correctly and not ignored in tests

Reply:
done

Reply:
reverted as doing that broke resource. i see plenty of resource with location/some other arg being ignored in import. these are url only params and are not read from api. only used to in URL. unless you have some other guidance/example there is nothing to add here.